doc/classes/Spatial.xml

```diff
@@ -110,14 +110,14 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns whether this node is set as Toplevel, ignoring its parent node transformations.
+				Returns whether this node is set as Toplevel (whether ignores its parent nodes transformations).
 			</description>
 		</method>
 		<method name="is_transform_notification_enabled" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Returns whether node sends notification that its transformation changed. Spatial will not propagate this by default.
+				Returns whether node sends notification that its transformation changed, global or local. Spatial will not propagate this by default.
```

Comment:
"whether *the* node".

A clearer sentence would be "Returns whether the node notifies about its global and local transformation changes. Spatial will not propagate this notification by default."

Reply:
Sounds good.